libs/langchain/tests/integration_tests/test_graph_store.py

```diff
@@ -1,283 +0,0 @@
-import math
```

Comment:
I'm a bit worried that we're deleting these tests. Are we sure they're running in LangChain? There is some benefit to keeping them here (I think) -- ideally, we'd have a way of running tests on the VectorGraphStore when we change `knowledge-store` that uses that rather than the released version...

Reply:
I think we should have other tests. Maybe in e2e-tests project.

Reply:
> Are we sure they're running in LangChain?

Indeed they're not. Maybe we'll keep them here for the moment. And move them to e2e-tests later.

Reply:
restored.